BashCompletion/README.md

```diff
@@ -0,0 +1,46 @@
+# Bash Completion
+
+## Installation
+
+    $ brew install bash-completion
+
+Bash completion will installed on `/usr/local/etc/bash_completion.d`
+
+So that this works, Add the following line to your ~/.bash_profile:
+
+```bash
+[ -f /usr/local/etc/bash_completion ] && . /usr/local/etc/bash_completion
```

Comment:
Is the `. /usr/local/...` right? With the dot followed by a whitespace. I've never seen it before and I can't find any information about it. What does it do?

Reply:
That's right.

this is info from brew command

```bash
$ brew info bash-completion
bash-completion: stable 1.3 (bottled)
Programmable completion for Bash 3.2
https://salsa.debian.org/debian/bash-completion
Conflicts with:
  bash-completion@2 (because Differing version of same formula)
/usr/local/Cellar/bash-completion/1.3_3 (189 files, 608.2KB) *
  Poured from bottle on 2018-12-08 at 00:25:02
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/bash-completion.rb
==> Caveats
Add the following line to your ~/.bash_profile:
  [ -f /usr/local/etc/bash_completion ] && . /usr/local/etc/bash_completion

Bash completion has been installed to:
  /usr/local/etc/bash_completion.d
```